samples/spring-boot/pom.xml

```diff
@@ -163,7 +164,7 @@
                                 <config>
                                     <spring-boot>
                                         <!-- authentic username of registry to be used for automatic push to registry-->
-                                        <!--<name>${image.user}/${project.artifactId}:${project.version}</name>-->
+                                        <name>${image.user}/${project.artifactId}:${project.version}</name>
```

Comment:
Please, revert as it was, (commented out)
`<!--<name>${image.user}/${project.artifactId}:${project.version}</name>-->`

Reply:
aah, my bad